extensions/amp-bodymovin-player/amp-bodymovin-player.md

```diff
@@ -0,0 +1,80 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-bodymovin-player"></a> `amp-bodymovin-player`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>An <code>amp-bodymovin-player</code> displays the AirBnB Bodymovin Player used to render the JSON generated by Adobe Effects.
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-bodymovin-player" src="https://cdn.ampproject.org/v0/amp-bodymovin-player-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fill, fixed, fixed-height, flex-item, nodisplay, responsive</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td><a href="https://ampbyexample.com/components/amp-bodymovin-player/"> Annotated code example for amp-bodymovin-player</a></td>
+  </tr>
+</table>
+
+[TOC]
+
+## Example
+
+The `width` and `height` attributes determine the aspect ratio of the player embedded in responsive layouts.
+
+Examples:
+
+```html
+  <amp-bodymovin-player
+    layout="fixed"
+    width="200" height="200"
+    data-animation-path="https://nainar.github.io/loader.json"
+    data-anim-loop="true"
+    data-name="ripple_1">
+  </amp-bodymovin-player>
+```
+
+## Attributes
+
+##### src
+
+The path to the exported Bodymovin animation object
+
+##### loop (optional)
+
+Whether the animation should be looping or not. `true` by default. Values can be: `true`|`false`|`number`
```

Comment:
`loop` or `loop=10` should work no need for true/false (binary attributes are so weird)

Reply:
bodymovin animations serve from the limited research I have done as:
* animation to signify action completed (you bought something)
* placeholder for loading
* a lot of keep the user 'occupied'
* other usecases that need just the 1 iteration

Given this I think looping the animation should be the default?